languages/python/sqlalchemy-oso/sqlalchemy_oso/session.py

```diff
@@ -291,41 +429,26 @@ def do_orm_execute(execute_state):
         if checked_permissions is None:
             return
 
-        def entities_in_statement(statement):
```

Comment:
I think we'll need to keep this method around for sqlalchemy <1.4. Perhaps we can add this old implementation to the pre-1.4 version of `all_entities_in_statement` in `sqlalchemy_utils.py`? (right now it raises an NotImplementedError)

@dhatch do you have any thoughts on this?